scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java

```diff
@@ -73,6 +73,9 @@ public OutputView(OutputController controller) {
 
         this.refreshButtonLabel = "Finished Tasks Output";
         this.refreshButtonTooltip = "Request fetching the logs for finished tasks of the entire Job, or for a selected Task";
+
+        this.notAuthorized = "You are not authorized to retrieve this job's output";
```

Comment:
This field does not look like a variable. Maybe you can remove this field at all and extract constant?